src/main/java/com/sourcery/gymapp/backend/sharedLinks/model/LinkAccessLogs.java

```diff
@@ -0,0 +1,25 @@
+package com.sourcery.gymapp.backend.sharedLinks.model;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "link_access_logs", schema = "shared_links")
+public class LinkAccessLogs extends BaseEntity {
+
+    @ManyToOne(cascade = CascadeType.ALL)
```

Comment:
this CascadeType means if we delete the logs about any links, the links will be deleted as well, probably we want to avoid that

Reply:
the CascadeTypy.ALL means that if one Log is deleted, the related Link from table Links will be deleted as well.
So, it would be better if there was no CascadeType specified 
Just @ManyToOne